scripts/remove/audiobookshelf.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Audiobookshelf
+
+__remove_audiobookshelf() {
+    echo_progress_start "Removing audiobookshelf apt sources"
+    systemctl stop audiobookshelf.service
+    sysctl disable audiobookshelf.service
+    apt remove audiobookshelf
+    rm -f /etc/apt/sources.list.d/audiobookshelf.list
+    apt update
+    apt autoremove
```

Comment:
We've got apt functions for these two, but I don't think you need to do the update here, just takes time. The apt tools will auto-update themselves as they're being called if necessary